Samples/All/BumbleRumble/Common/NetworkManager.h

```diff
@@ -67,6 +73,7 @@ class NetworkManager : public Manager
     Party::PartyLocalUser* m_localUser;
     Party::PartyLocalChatControl* m_localChatControl;
     bool m_partyInitialized;
+	bool m_enableCognitiveServices;

```

Comment:
strange indenting?

Reply:
Visual studio defaults to using tabs instead of spaces. I will fix this up.